src/tests/Navbar.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Navbar from '../components/Navbar';
+import store from '../redux/store';
+
+test('Check navbar component rendered correctly', () => {
+  const tree = render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Navbar />
+      </BrowserRouter>
+    </Provider>,
+  );
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Good work writing snapshots tests so far. However, you only have 4 test cases and 1 of them is failing. Also, as per project requirements. Please make sure to test several components and pure functions related to Redux as well as integration tests of your web app. 🙏 

![MINGW64__c_Users_mbakr_OneDrive_Desktop_metrics-webapp 9_9_2023 12_49_30 PM](https://github.com/JoyTaribagshaw/metrics-webapp/assets/47625407/82233325-91c6-404a-b03d-963b185b1926)

Reply:
Thank you for your review, I have implemented the changes.

![image](https://github.com/JoyTaribagshaw/metrics-webapp/assets/111312788/b6b47d95-13e5-49ab-a3fe-96c04383181b)